src/main/kotlin/no/nav/familie/ks/sak/api/BrevController.kt

```diff
@@ -52,11 +61,51 @@ class BrevController(
         )
 
         return brevService.hentForhåndsvisningAvBrev(
-            behandlingId = behandlingId,
-            manueltBrevDto = manueltBrevDto,
+            manueltBrevDto = manueltBrevDto.utvidManueltBrevDtoMedEnhetOgMottaker(behandlingId, personopplysningGrunnlagService, arbeidsfordelingService),
+        ).let { Ressurs.success(it) }
+    }
+
+    @PostMapping(path = ["/fagsak/{fagsakId}/forhåndsvis-brev"])
+    fun hentForhåndsvisningPåFagsak(
+        @PathVariable fagsakId: Long,
+        @RequestBody manueltBrevDto: ManueltBrevDto,
+    ): Ressurs<ByteArray> {
+        logger.info(
+            "${SikkerhetContext.hentSaksbehandlerNavn()} henter forhåndsvisning av brev på fagsak $fagsakId " +
+                "for mal: ${manueltBrevDto.brevmal}",
+        )
+        tilgangService.validerTilgangTilHandlingOgFagsakForPerson(
+            minimumBehandlerRolle = BehandlerRolle.SAKSBEHANDLER,
+            handling = "hente forhåndsvisning brev",
+            personIdent = manueltBrevDto.mottakerIdent,
+            event = AuditLoggerEvent.ACCESS,
+        )
+
+        return brevService.hentForhåndsvisningAvBrev(
+            manueltBrevDto = manueltBrevDto.leggTilEnhet(arbeidsfordelingService),
         ).let { Ressurs.success(it) }
     }
 
+    @PostMapping(path = ["/fagsak/{fagsakId}/send-brev"])
+    fun sendBrevPåFagsak(
+        @PathVariable fagsakId: Long,
+        @RequestBody manueltBrevDto: ManueltBrevDto,
+    ): ResponseEntity<Ressurs<MinimalFagsakResponsDto>> {
+        logger.info("${SikkerhetContext.hentSaksbehandlerNavn()} genererer og sender brev på fagsak $fagsakId: ${manueltBrevDto.brevmal}")
+        tilgangService.validerTilgangTilHandling(
+            minimumBehandlerRolle = BehandlerRolle.SAKSBEHANDLER,
+            handling = "sende brev",
```

Comment:
Trenger man ikke ett AuditLoggerEvent?

Reply:
Ikke på validerTilgangTilHandling ser det utsom i KS

Reply:
Slik jeg da leser koden, så blir det da ikke auditlogget.

Reply:
Jeg kan sikkert endre på det, men greit at det gjøres i en separat PR?
Det er veldig mange metoder som bruker validerTilgangTilHandling allerede, på samme måte som `verifiserHarTilgangTilHandling` blir brukt i ba, og der er det heller ingen logging.

Reply:
Neida. Prøver bare å forstå forskjellen siden dette er litt ukjent for meg.

Reply:
Helt forståelig 😄 
Kommer en PR på det snart!